x-pack/plugins/timelines/public/components/t_grid/body/helpers.test.tsx

```diff
@@ -122,12 +123,22 @@ describe('helpers', () => {
 
     test('it returns the expected results when each column has a corresponding entry in `columnHeaders`', () => {
       expect(mapSortingColumns({ columns, columnHeaders })).toEqual([
-        { columnId: 'kibana.rac.alert.status', columnType: 'string', sortDirection: 'asc' },
-        { columnId: 'kibana.rac.alert.start', columnType: 'date', sortDirection: 'desc' },
+        {
+          columnId: 'kibana.rac.alert.status',
+          columnType: 'string',
+          esTypes: [],
+          sortDirection: 'asc',
+        },
+        {
+          columnId: 'kibana.rac.alert.start',
+          columnType: 'date',
+          esTypes: ['date'],
+          sortDirection: 'desc',
+        },
       ]);
     });
 
-    test('it defaults to a `columnType` of `text` when a column does NOT has a corresponding entry in `columnHeaders`', () => {
+    test('it defaults to a `columnType` of empty string when a column does NOT has a corresponding entry in `columnHeaders`', () => {
```

Comment:
```suggestion
    test('it defaults to a `columnType` of empty string when a column does NOT have a corresponding entry in `columnHeaders`', () => {
```

I know the typo was there before, so feel free to ignore

Reply:
> I know the typo was there before, so feel free to ignore

Appreciate your attention to detail @janmonschke! Per our offline team discussion, i'll change the `has` to `have` in this test description in a follow-up PR.